docker/ubuntu-16.04-cuda8-cudnn6-all-options/Dockerfile

```diff
@@ -0,0 +1,93 @@
+FROM nvidia/cuda:8.0-cudnn6-devel-ubuntu16.04
+MAINTAINER Aaron Markham <aaronmarkham@fb.com>
+
+# caffe2 install with gpu support
+
+########## REQUIRED DEPENDENCIES ################
+RUN apt-get update
```

Comment:
This should be combined with the `RUN` below, see the Docker recommendations:
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run